src/form-builder/RadioGroup.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { FieldProps } from './types';
+import {
+  VaRadio,
+  VaRadioOption,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { useField, FieldHookConfig } from 'formik';
+import { chainValidations, required } from '../utils/validation';
+
+type RadioGroupProps = FieldProps<string> & {
```

Comment:
Do we want to just add these types to the types.ts file? Also should we be using `interface` instead of `type`?